### PR TITLE
chore(i18n,learn): include 2 translated certs in Portuguese curriculum

### PR DESCRIPTION
**Pull request by DanielRosa74**
Added College algebra and Project Euler to the translated certifications.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by raisedadead** [comment]
Hey @ DanielRosa74, can you take a look at the recommended contributing guidelines in the checklist on the PR? 

This can't be merged as long as those aren't addressed.

File: config/superblock-order.ts [hunk 1]
```diff
@@ -353,7 +353,7 @@ export const superBlockOrder: SuperBlockOrder = {
       SuperBlocks.DataAnalysisPy,
       SuperBlocks.InfoSec,
       SuperBlocks.MachineLearningPy,
-      SuperBlocks.CollegeAlgebraPy
+      SuperBlocks.CollegeAlgebraPy
```
Comment (Sboonny) [inline config/superblock-order.ts:356]: To please prettier

```suggestion
      SuperBlocks.CollegeAlgebraPy
```

File: config/superblock-order.ts [hunk 2]
```diff
@@ -369,17 +369,16 @@ export const superBlockOrder: SuperBlockOrder = {
           SuperBlocks.DataAnalysisPy,
           SuperBlocks.InfoSec,
           SuperBlocks.MachineLearningPy,
-          SuperBlocks.CodingInterviewPrep
+          SuperBlocks.CodingInterviewPrep,
+          SuperBlocks.CollegeAlgebraPy,
```
Comment (Sboonny) [inline config/superblock-order.ts:373]: The test will pass if you swap these values.

```suggestion
          SuperBlocks.CollegeAlgebraPy,
          SuperBlocks.CodingInterviewPrep,
```

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by Sembauke — APPROVED** [review]

**Pull request merged**